### ISSUE TITLE
Splitter | Overflow issue

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#15959]
- Overflow is visible due to missing `overflow: hidden; `style in `.p-splitter-panel `element.

![Screenshot 2024-07-04 at 14 28 32](https://github.com/primefaces/primeng/assets/69278826/63fff003-a66a-45bb-a780-eb3ad357dc4a)

**Comment by randy9527** [comment]
Adding `overflow: hidden;` to splitter panel introducing another issue:

If there's a cascade select hosted in vertical splitter panel, the cascaded selections will be blocked by panel:
With `overflow: hidden;`:
![image](https://github.com/primefaces/primeng/assets/88707662/74f3c403-e374-416b-a832-3fe0d0903822)
Without `overflow: hidden;`:
![image](https://github.com/primefaces/primeng/assets/88707662/edbcf391-e131-40db-865b-de85fe3f70d7)
It can be easily reproduced by mixing splitter and cascade select demos.

If there's another approach to fix or any workaround?

**Comment by cetincakiroglu** [comment]
Hi @randy9527 ,

Overflow should be hidden since it's out of the boundaries of panel 1. You can override the CSS and make overflow: scroll or whatever you need as a workaround.
